nuxeo-jsf/nuxeo-platform-webapp-base/src/main/java/org/nuxeo/ecm/webapp/directory/VocabularyTreeActions.java

```diff
@@ -85,7 +86,7 @@ public VocabularyTreeNode get(String vocabularyName, boolean displayObsoleteEntr
             // return cached model
             return treeModel;
         }
-        DirectoryService directoryService = Framework.getService(DirectoryService.class);
+        DirectoryService directoryService = DirectoryHelper.getDirectoryService();
```

Comment:
Why add a dependency to `DirectoryHelper` and not just keep the current code?

Reply:
+1